lib_bgp_data/utils/utils.py

```diff
@@ -352,3 +354,22 @@ def replace_line(path, prepend, line_to_replace, replace_with):
     for line in fileinput.input(path, inplace=1):
         line = line.replace(*lines)
         sys.stdout.write(line)
+
+def send_email(subject, body):
```

Comment:
Do these environment variables persist? For example, Cameron just rebooted the server. Do these emails still work?